doc/manuals/installation.rst

```diff
@@ -0,0 +1,93 @@
+.. _installation:
+
+.. |telesculptor_install_1| image:: /images/telesculptor_install_1.png
+   :width: 49 %

```

Comment:
We don't need to force these to be half page width and make them fit two across.  That was just done to make them fit better on the page in Word.  Here they can be full size images (or maybe 80% width if full size looks too large) and they could be one on each line.  Fitting two on a line doesn't seem to work reliably in RST.

Reply:
Inline images with captions aren't permitted in RST. I was trying to find a workaround but none seem to exist.

Reply:
They don't need to be inline, you can do it this way.  My point was that they don't need to be 49% wide.  We could bump that up to may be 80% so each is on a new line.  The 49% sometimes puts image side-by-side but sometimes doesn't depending on the browser.  They don't need to be side-by-side in the HTML documentation so we don't need to force it.